brewchat/src/main/java/com/example/brewchat/ChatService.java

```diff
@@ -75,7 +81,12 @@
 
     static final String TAG = ChatService.class.toString();
 
-    protected User currentUser=null;
+    protected User currentUser = null;
+
+    // Need to keep track of the JID of the chat rooms, but don't want to expose that outside of this ChatService
```

Comment:
I was thinking that. That'll be a challenge of it's own though, how old does "recent" entail?